core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/components/dot-content-aside-workflow/dot-content-aside-workflow.component.scss

```diff
@@ -0,0 +1,37 @@
+@use "variables" as *;
+
+.content-aside__workflow {
+    display: flex;
+    gap: $spacing-3;
+    padding: $spacing-1 $spacing-4;
+}
+
+.content-aside__workflow-column {
+    display: flex;
+    flex: 1 0 0;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: $spacing-1;
+    overflow: hidden;
+}
+
+.workflow-column__title {
+    font-weight: $font-weight-medium-bold;
+}
+
+.item__description {
+    color: $color-palette-gray-800;
+    margin: 0;
+}
+
+.workflow-column__title,
+.workflow-column__description {
+    color: $black;
+    height: 32px;
+    font-size: 14px;
+    line-height: $line-height;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
```

Comment:
Check if the mixin I mentioned before works here too

Reply:
Done!